optimizer_wrapper.rb

```diff
@@ -690,11 +690,25 @@ def self.route_details(vrp, route, vehicle)
     details
   end
 
+  def self.provide_original_ids(data)
```

Comment:
Shall we initialize the original id when the vrp is constructed for the first time ? And use this value when the result is under construction ?